domain/application/state/application.go

```diff
@@ -828,31 +828,33 @@ WHERE application_uuid = $applicationScale.application_uuid
 	return errors.Capture(err)
 }
 
-// CloudServiceAddresses returns the addresses of the cloud service for the
+// GetCloudServiceAddresses returns the addresses of the cloud service for the
 // specified application, returning an error satisfying
 // [applicationerrors.ApplicationNotFoundError] if the application doesn't
 // exist.
-func (st *State) CloudServiceAddresses(ctx context.Context, applicationName string) (network.SpaceAddresses, error) {
+func (st *State) GetCloudServiceAddresses(ctx context.Context, applicationName string) (network.SpaceAddresses, error) {
 	db, err := st.DB()
 	if err != nil {
 		return nil, errors.Capture(err)
 	}
-	var addresses []ipAddress
+	var addresses []spaceAddress
 	ident := dbUUID{}
 	queryCloudServiceAddressesStmt, err := st.Prepare(`
 SELECT    
-    ip.address_value AS &ipAddress.address_value,
-    ip.config_type_id AS &ipAddress.config_type_id,
-    ip.type_id AS &ipAddress.type_id,
-    ip.origin_id AS &ipAddress.origin_id,
-    ip.scope_id AS &ipAddress.scope_id,
-    ip.device_uuid AS &ipAddress.device_uuid
+    ip.address_value AS &spaceAddress.address_value,
```

Comment:
`select &spaceAddress.*`

Reply:
I had to use all the separate fields because I was using the ip address uuid as output. But it turns out I don't need it anymore so I can now do `&spaceAddress.*`. 
Thanks!